packages/router-history/test/index.test.js

```diff
@@ -0,0 +1,55 @@
+import RouterHistory from  '../src/index';
+
+const baseUrl = '/tests';
+
+const history = {
+    location: {
+      pathname: `${baseUrl}/posts/postId1`,
+      search: '?op=create&payload={"title":"Default title"}'
+    },
+    push: () => {}
+};
+
+describe('RouterHistory', () => {
+  it('should getRoutes return expected results', () => {
+    const router = new RouterHistory({baseUrl, history});
+    expect(router.getRoutes()).toEqual(['posts', 'postId1']);
+  });
+  
+
+  it('should getParams return expected results', () => {
+    const router = new RouterHistory({baseUrl, history});
+    expect(router.getParams()).toEqual({op: 'create', payload: {title: 'Default title'}});
```

Comment:
I'm thinking we should separate `getParams` to more specific methods like `getOperator`, `getPayload()` and `getFilter()` because it's meaningless if we simply expose all properties with a large object.

Reply:
@abz53378:  I'm not sure what kind of params will be included (and the way be used) in the router. What do you think?